src/rabbit_authz_backend.erl

```diff
@@ -45,9 +45,11 @@
 %% false
 %% {error, Error}
 %%     Something went wrong. Log and die.
+%% {error_message, Message}
+%%     Report Message to a client using protocol error and die.
 -callback check_vhost_access(rabbit_types:auth_user(),
                              rabbit_types:vhost(), rabbit_net:socket()) ->
-    boolean() | {'error', any()}.
+    boolean() | {'error', any()} | {'error_message', string()}.
```

Comment:
I see from the other PR that both `{error, _}` and '{error_message, _}` result in a protocol error.